### DISCUSSION TITLE
discord email verification error

### DISCUSSION THREAD

**Discussion by brah-mcdude** [akkadotnet/akka.net#6970] [General]
![image](https://github.com/akkadotnet/akka.net/assets/77924970/03021ede-e2ef-4934-b58a-2395fcfb7726)

**Comment by Aaronontheweb** [comment]
Have you tried resetting your password?

**Comment by brah-mcdude** [comment]
![image](https://github.com/akkadotnet/akka.net/assets/77924970/d6752089-f7eb-42bf-8211-5c6a05e758c2)

**Comment by brah-mcdude** [comment]
> Have you tried resetting your password?

i can not log in.

**Reply by Aaronontheweb** [reply]
Don't think I can help you then - this is a Discord.com level issue, not a problem with our server.

**Comment by brah-mcdude** [comment]
from my experience (not my first account to be closed by discord for no reason at all) it is a dead end. i'll open an issue about my akka issue.

**Reply by Aaronontheweb** [reply]
Sorry about that @brah-mcdude
